com.unity.netcode.gameobjects/Runtime/Messaging/MessagingSystem.cs

```diff
@@ -385,6 +385,8 @@ private bool CanSend(ulong clientId, Type messageType, NetworkDelivery delivery)
                         continue;
                     }
 
+                    Debug.Log($"Sending {typeof(TMessageType)} to {clientId}");
```

Comment:
I think this got added by mistake? As it's quite spammy I guess?

Reply:
Good call, removed.